### ISSUE TITLE
Game should remember when you were last in Satcom

### ISSUE THREAD

**Issue by narczi** [colobot/colobot#904]
I open Satcom, let's say I want to read up on Radar function, scroll down, click on Radar... okay, I'm done reading. I go back in game, do something but I forget what a certain parameter passed to Radar did, so I open Satcom... and I have to navigate to Radar function again... and again, and again, and again. It would be nice if Satcom remembered what page you had open + scroll position between opening and closing it.

**Comment by melex750** [comment]
Click the keyword help button to open SatCom...
![](https://www.dropbox.com/s/hhqoe4ujy6kiqsq/keywordHelp.png?dl=1)

**Comment by FreeER** [comment]
@melex750 true, that is possible but having it just be exactly where we were previously (including the line half way down the page not the top) removes just a little bit more friction.

To me it's just a nice polishing thing like an ebook taking you directly to the page you were reading the last time you had a book open rather than just the last chapter you were reading or table of contents, sure it'd be a lot more aggravating for ebooks because that's very much part of what they do but it's not strictly necessary for their purpose. Like clicking the help button to open SatCom :)
